internal/backends/collection.go

```diff
@@ -76,8 +76,11 @@ func CollectionContract(c Collection) Collection {
 
 // QueryParams represents the parameters of Collection.Query method.
 type QueryParams struct {
-	Filter        *types.Document
-	Sort          *types.Document
+	Filter *types.Document
+	Sort   *types.Document
+
+	DisablePushdown bool
```

Comment:
We use it only for checking other parameters, but that should be in the caller. Let's remove it from there

Reply:
That's not true. It is used by backend to disable capped collection sort pushdown here: https://github.com/FerretDB/FerretDB/pull/3793/files#diff-ed1b75cf02347cc3668ac3770a3e8b1197504bd78835652b97516604eb9f62b8R99

Reply:
Well, yeah, you added it in this PR. Let's remove that field and its usage